pyomo/contrib/trustregion/examples/ipopt.opt

```diff
@@ -1,5 +0,0 @@
-linear_solver mumps
```

Comment:
These options are not the default of IPOPT and may help with convergence (reduced tolerance and bound push). Is it worth it to remove completely? Maybe needed in the documentation at least.

Reply:
I don't know if it's necessary to have an example IPOPT configuration in the Trust Region. Plus, in the newer version of the algorithm (Yoshio & Biegler), Yoshio doesn't change IPOPT configurations.

Basically, I have no horse in this race. If the consensus is that we should keep this, I'm fine with that.

Reply:
One should not force solver options if there is no strong motivation for it. As long as the tests pass, this can be removed.